src/screens/IdentityNew.js

```diff
@@ -62,7 +62,7 @@ function IdentityNew({ accounts, navigation }) {
 	const onSeedTextInput = inputSeedPhrase => {
 		setSeedPhrase(inputSeedPhrase);
 		const addressGeneration = () =>
-			brainWalletAddress(inputSeedPhrase)
+			brainWalletAddress(inputSeedPhrase.trimEnd())
```

Comment:
Can `trimEnd` be removed here and below if `trim` is used in `onPress` above? If so it would be better to do it in just the one place.